tests/rules/taint_async.yaml

```diff
@@ -0,0 +1,16 @@
+rules:
+  - id: test
+    severity: WARNING
+    languages:
+      - python
+    message: Test
+    mode: taint
+    pattern-sources:
```

Comment:
you should use the new syntax and move the test in tests/rules_v2/

Reply:
Do we have a date for the new syntax? I'm a bit unsure about doing that, we have a translator from "current syntax" to "new/future syntax", so it's easy to migrate tests later on. But if any detail of the "new syntax" changes in the meantime, that is something we may end up having to fix manually.

Reply:
I think it's good to dogfood. Part of the OSS vs Pro document is looking towards migrating also to the new syntax.

Reply:
Let's discuss via Slack.